src/main/java/org/broadinstitute/hellbender/engine/filters/MappingQualityReadFilter.java

```diff
@@ -14,18 +14,28 @@
 public final class MappingQualityReadFilter extends ReadFilter implements Serializable {
     private static final long serialVersionUID = 1L;
 
-    @Argument(fullName="minimumMappingQuality", shortName="minimumMappingQuality", optional=true)
+    @Argument(fullName="minimumMappingQuality", shortName="minimumMappingQuality", doc = "Minimum mapping quality to keep (inclusive)", optional=true)
     public int minMappingQualityScore = 10;
 
+    @Argument(fullName="maximumMappingQuality", shortName="maximumMappingQuality", doc = "Maximum mapping quality to keep (exclusive)", optional=true)
```

Comment:
Looks like the maximum is actually *inclusive* (`<=`), not exclusive (`<`) as the doc says. Can you update the doc here? I think we should keep the comparison inclusive, as an exclusive comparison would be too confusing.